web3/main.py

```diff
@@ -223,38 +241,38 @@ def solidityKeccak(cls, abi_types, values):
         ))
         return cls.keccak(hexstr=hex_string)
 
-    def isConnected(self):
+    def isConnected(self) -> bool:
         return self.provider.isConnected()
 
-    def is_encodable(self, _type, value):
+    def is_encodable(self, _type: TypeStr, value: Any) -> bool:
         return self.codec.is_encodable(_type, value)
 
     @property
-    def ens(self):
+    def ens(self) -> Union[ENS, Empty]:
```

Comment:
I'm not sure if there's a better way to handle this, since it should always be returning only `ENS` - but since `web3.ens` is set to a default `Empty` type in the constructor args, this seems like the only way to make mypy happy.

Reply:
I think you could fix this using a `cast` right?